interface/patient_tracker/patient_tracker.php

```diff
@@ -172,7 +259,20 @@ function openNewTopWindow(newpid,newencounterid) {
 <?php } ?>
 
 <table border='0' cellpadding='1' cellspacing='2' width='100%'>
-
+ <tr>
+     <td colspan="12">
+         <b><small>
+             <?php
+             $statuses_output =  xlt('Total patients')  . ':' . $appointments_status['count_all'];
+             unset($appointments_status['count_all']);
+             foreach($appointments_status as $status_symbol => $count){
+                 $statuses_output .= " | " . xlt($statuses_list[$status_symbol])  .":" . $count;
```

Comment:
If you are bringing back the symbols, then use this:
``````
$statuses_output .= " | " . text(getListItemTitle('apptstat',$statuses_list[$status_symbol]))  .":" . $count;
``````
If you are bringing back the titles, then use this:
``````
$statuses_output .= " | " . text(xl_list_label($statuses_list[$status_symbol]))  .":" . $count;
``````